.github/workflows/macos-integration-tests.yml

```diff
@@ -2,7 +2,17 @@ name: macOS Integration Tests
 
 on:
   push:
+    paths:
+      - 'sdk/python/packages/flet/src/**'
+      - 'sdk/python/packages/flet/integration_tests/**'
   pull_request:
+    paths:
+      - 'sdk/python/packages/flet/src/**'
+      - 'sdk/python/packages/flet/integration_tests/**'
+
+concurrency:
+  group: macos-integration-tests-${{ github.ref }}
```

Comment:
What does `github.ref` mean here?

Reply:
OK, so it's per branch. Understood.

Reply:
I modified the group name to prevent duplicate runs when pushing a commit to a branch which has an opened PR.
<img width="454" height="71" alt="image" src="https://github.com/user-attachments/assets/1c324e33-ff43-4786-a26f-30bf875680ca" />

Now in such cases, only the PR run will take the lead, and the branch run will be cancelled.